packages/design-tokens/src/scss/generated/_borders.scss

```diff
@@ -0,0 +1,7 @@
+// Generated Borders from Supernova. Do not edit manually.
+$border-style-0: none !default;
+$border-style-100: solid !default;
+$border-style-200: dashed !default;
+$border-width-0: 0 !default;
+$border-width-100: 1px !default;
+$border-width-200: 2px !default;
```

Comment:
If design tokens are managed in Supernova and all changes should always come from designers, maybe we don't need `!default` flags at all—is it right?

Reply:
I don't think so. You still have to be able to overwrite defaults even when you don't use supernova.